### ISSUE TITLE
Ever got this running on a raspberry pi? 

### ISSUE THREAD

**Issue by craigvphillips** [jsclayton/prometheus-plex-exporter#15]
I can build fine but it won't run because it complains of ENV variables not being set, but they are set. 

Can't get dockerfile to build unless I make a bunch of changes, and it always gives me docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/app/prometheus-plex-exporter": permission denied: unknown.

**Comment by craigvphillips** [comment]
close

**Comment by MasterEvarior** [comment]
@craigvphillips Did you ever manage to get it running?

**Comment by craigvphillips** [comment]
Yes, it's just go, I was making it way harder than it should have been.

**Comment by MasterEvarior** [comment]
Can you explain to me how you did it? I getting the same error as you are but have not yet found a solution.

**Comment by craigvphillips** [comment]
Absolutely, when I get home I can get my docker file and commands and I will reply to you.

**Comment by MasterEvarior** [comment]
Thank you very much :smiley:

**Comment by craigvphillips** [comment]
well crap, I don't really know what I'm doing in go, but essentially you have to clone the repo and build a go executable on your raspberry pi so it's the right architecture. However, I'm getting an error now. 

pi@raspberrypi:~/promplex/prometheus-plex-exporter $ go build
can't load package: package github.com/grafana/plexporter: unknown import path "github.com/grafana/plexporter": cannot find module providing package github.com/grafana/plexporter

Definitely didn't have that error when I did it on my other one. I will also say that the grafana dashboard doesn't work very well for the watched movies, hoping someone smarter than me can fix it one day. I'll ask some go people tomorrow how to fix that error.
